CODEOWNERS

```
@@ -27,6 +27,7 @@
 /.vscode/                                 @kylebonnici
 
 # Applications
+/applications/asset_traker_template/README.rst @nrfconnect/ncs-cia-doc
```

```suggestion
/applications/asset_tracker_template/README.rst @nrfconnect/ncs-cia-doc
```
The `c` is missing also in the path of the new file